Cargo.toml

```diff
@@ -157,6 +157,7 @@ name = "ring"
 cfg-if = { version = "1.0.0", default-features = false }
 getrandom = { version = "0.2.10" }
 untrusted = { version = "0.9" }
+serde = { version = "1.0.203", optional = true, features = ["derive"] }
```

Comment:
Hi, I don't want a serde dependency. We need a serde-free API.

Reply:
Hi, serde removed